CONTRIBUTING.md

```diff
@@ -0,0 +1,246 @@
+Larger changes deserve a comment in the main thread, or a pull request to the original author's branch (but mention that you've made one).
+When in doubt about a matter of architectural philosophy, refer back to [*What we're trying to build*](https://github.com/bevy-engine/bevy/blob/main/CONTRIBUTING.md#what-were-trying-to-build) for guidance.
+
+Once you're happy with the work and feel you're reasonably qualified to assess quality in this particular area, leave your `Approved` review on the PR so we can mark it as `ready-for-cart`.
+
+There are a two main places you can check for new work to review:
+
+1. Pull requests on `bevy` and the `bevy-website` repos.
+2. [RFCs](https://github.com/bevyengine/rfcs), which need extensive thoughtful community input on their design.
+
+Official focus areas and work done by @cart go through this review process as well.
+Not even our project lead is exempt from reviews and RFCs!
+By giving feedback on this work (and related supporting work), you can help us make sure our releases are both high-quality and timely.
+
+Finally, if nothing brings you more satisfaction than seeing every last issue tagged and all resolved issues closed, feel free to message @cart for a Bevy org role to help us keep things tidy.
+As discussed in [*How we're organized*](https://github.com/bevy-engine/bevy/blob/main/CONTRIBUTING.md#how-were-organized), this is not intended to have a high bar.
+
+### Contributing your own ideas
+
+Bevy is actively open to new ideas and serious contributions from outside community members.
+If you're new to Bevy, here's the workflow we use:
+
+1. Fork the `bevyengine/bevy` repository on GitHub, you'll need to create a GitHub account if you don't have one already.
+2. Make your changes in a local clone of your fork, typically in its own new branch.
+3. For a higher chance of CI passing the first time, consider locally running `cargo run -p ci`. You can run the commands manually:
+    1. `cargo fmt --all -- --check` (remove `--check` to let the command fix found problems)
+    2. `cargo clippy --workspace --all-targets --all-features -- -D warnings -A clippy::type_complexity`
+    3. `cargo test --all-targets --workspace`
+4. When working with Markdown (`.md`) files, Bevy's CI will check markdown files like this one using [markdownlint](https://github.com/DavidAnson/markdownlint).
+To locally lint your files using the same workflow as our CI:
+   1. Install [markdownlint-cli](https://github.com/igorshubovych/markdownlint-cli).
+   2. Run `markdownlint -f -c .github/linters/.markdown-lint.yml .` in the root directory of the Bevy project.
+5. Push your changes to your fork and open a Pull Request.
+6. If you're a first time contributor to this repo, @cart [will need to manually trigger CI for your PR](https://github.blog/changelog/2021-04-22-github-actions-maintainers-must-approve-first-time-contributor-workflow-runs/). Feel free to ping him for this.
+7. Respond to any CI failures or review feedback.
```

Comment:
```suggestion
7. Respond to any CI failures or review feedback. CI failures must be fixed, review feedback should be answered but you are not obligated to follow them, use your best judgement.
```
not completely happy about my sentence here. I mean that while CI is mandatory, reviews from other members don't have the same weight and people are free to follow them or not. they may even get contradictory reviews!

Reply:
I like the sentiment. Pushing a commit with better wording for you.